alacritty_terminal/src/vi_mode.rs

```diff
@@ -756,4 +746,40 @@ mod tests {
         cursor = cursor.motion(&mut term, ViMotion::WordLeft);
         assert_eq!(cursor.point, Point::new(Line(0), Column(0)));
     }
+
+    fn history_term() -> Term<()> {
+        let mut term = term();
+        term.grid_mut().clear_viewport();
+        term
+    }
```

Comment:
Not sure this is helpful as a helper function. Probably easier to just inline it, since the behavior is a little more clear then.

Reply:
> Scrolling with the mouse. This is definitely still required.

I see. It is missing insight for me. I readd the code and add some tests to realize this regression.

> Not sure this is helpful as a helper function. Probably easier to just inline it, since the behavior is a little more clear then.

You're right. I remove this function and make it inline.